core-web/apps/dotcms-ui/src/app/view/components/dot-iframe-dialog/dot-iframe-dialog.component.ts

```diff
@@ -98,6 +101,7 @@ export class DotIframeDialogComponent implements OnChanges, OnInit {
      * @memberof DotIframeDialogComponent
      */
     onDialogHide(): void {
+        this.location.back();
```

Comment:
We use this component for a lot of things... this might affects other places where we use it... we need to tests this.

Reply:
Testing it!